src/routes/living-form/containers/NeuburgForm.js

```diff
@@ -0,0 +1,216 @@
+            name='formData.accommodation.moveInDate'
+          /></StdCol>
+        </Row>
+
+        <h3>Mietkosten</h3>
+        <Row>
+          <StdCol><TextInput name='formData.costs.baseRent' label='Grundmiete monatlich' validations={[required]}
+                             type='number' additionalLabel='Ohne Nebenkosten, Garage und Heizung'
+                             InputProps={{startAdornment: <InputAdornment position='start'>€</InputAdornment>}}
+          /></StdCol>
+          <StdCol><TextInput name='formData.costs.runningCosts' label='Nebenkosten monatlich' validations={[required]}
+                             type='number'
+                             InputProps={{
+                               startAdornment: <InputAdornment position='start'>€</InputAdornment>
+                             }} /></StdCol>
+        </Row>
+        <Row>
+          {map(runningServices, (label, key) => (
+            <NarrowCol key={key}><ArrayCheckbox name='formData.costs.ofRunningServices' label={label}
+                                                value={key} /></NarrowCol>
+          ))}
+        </Row>
+        <Row><WideCol><SingleCheckbox name='formData.costs.hotWaterInHeatingCosts'
+                                      label='Warmwasser in Heizung enthalten' /></WideCol></Row>
+        <Row><StdCol><TextInput name='formData.costs.additionalCosts' label='Zusatzkosten monatlich'
+                                validations={[required]}
+                                type='number'
+                                InputProps={{
+                                  startAdornment: <InputAdornment position='start'>€</InputAdornment>
+                                }} /></StdCol></Row>
+        <Row>
+          {map(additionalServices, (label, key) => (
+            <NarrowCol key={key}><ArrayCheckbox name='formData.costs.ofAdditionalServices' label={label}
+                                                value={key} /></NarrowCol>
+          ))}
+        </Row>
+        <h3>Gültigkeitsdauer und Datenschutz</h3>
+        <Row>
+          <StdCol>
+            <SelectInput name='duration' label='Gültigkeitsdauer des Angebots' validations={[required]}>
+              <MenuItem value={3}>3 Tage</MenuItem>
+              <MenuItem value={7}>7 Tage</MenuItem>
+              <MenuItem value={14}>14 Tage</MenuItem>
+              <MenuItem value={30}>30 Tage</MenuItem>
+            </SelectInput>
+          </StdCol>
+        </Row>
+        <Row>
+          <WideCol>
+            <SingleCheckbox name='agreedToDataProtection' label='Ich akzeptiere die Datenschutzerklärung:'
+                            validations={[needsAcceptance]} />
+            <LawParagraph>
+              Ich willige ein, dass der Landkreis Neuburg-Schrobenhausen und die Tür an
+              Tür - Digital Factory gGmbH meine personenbezogenen Daten zum Zwecke
+              der Wohnraumakquise für Anerkannte Flüchtlinge und bleibeberechtigte
```

Comment:
'anerkannt' should be lowercase

Reply:
Right, that jumped into my face as well, but I thought I wouldn't touch it, since it comes directly from neuburg. But I will change it 👍